BREAKING.md

```diff
@@ -90,7 +90,7 @@ The Container and RelativeLoader classes in `@fluidframework/container-loader` h
 
 ### BlobAggregationStorage and SnapshotExtractor deprecated
 
-The Container and RelativeLoader classes in `@fluidframework/driver-utils` have been deprecated and will be removed in
```

Comment:
Fixing this typo is not related to the script fix, but is related to the breaking change I'm trying to make.  Just including it here for convenience.